libusb/os/windows_usbdk.c

```diff
@@ -588,8 +596,18 @@ static int usbdk_do_bulk_transfer(struct usbi_transfer *itransfer)
 	case TransferSuccessAsync:
 		break;
 	case TransferFailure:
-		usbi_err(TRANSFER_CTX(transfer), "ReadPipe/WritePipe failed: %s", windows_error_str(0));
-		return LIBUSB_ERROR_IO;
+		error_code = GetLastError();
+		switch (error_code) {
+			case ERROR_FILE_NOT_FOUND:
```

Comment:
Thank you for combing through the code base to find the error conditions that lead to `LIBUSB_ERROR_NO_DEVICE`. Note: We don't use the usbdk backend but it's probably better to have this in here anyway for completeness.